solr/test-framework/src/java/org/apache/solr/util/RandomNoReverseMergePolicyFactory.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.solr.util;
+
+import org.apache.lucene.index.MergePolicy;
+import org.apache.solr.index.MergePolicyFactory;
+import org.apache.solr.index.MergePolicyFactoryArgs;
+
+/** A {@link MergePolicyFactory} for {@link RandomMergePolicy}. */
+public final class RandomNoReverseMergePolicyFactory extends MergePolicyFactory {
```

Comment:
I don't understand how this prevents "ReverseMergePolicy"

Reply:
Here we go 
https://github.com/apache/solr/pull/2289/files/d24bfff1fe1aaa11a666c8f7188895ce23eed128#diff-681b5f32ae2e1f988d11d5112305b5c71e08b3c3abee5879db27a9b91d63b398R46

Reply:
@dsmiley thanks for the guidance. I'll check it out.